File: ueueo-framework/ueueo-blob-storing/src/main/java/Abp/BlobStoring/BlobProviderGetArgs.cs
using System.Threading;
using JetBrains.Annotations;

namespace Volo.Abp.BlobStoring;

public class BlobProviderGetArgs : BlobProviderArgs
{
    public BlobProviderGetArgs(
        @Nonnull String containerName,
        @Nonnull BlobContainerConfiguration configuration,
        @Nonnull String blobName,
        CancellationToken cancellationToken = default)
        : base(
            containerName,
            configuration,
            blobName,
            cancellationToken)
    {
    }
}

File: ueueo-framework/ueueo-blob-storing/src/main/java/com/ueueo/blobstoring/BlobProviderGetArgs.java
package com.ueueo.blobstoring;

import com.ueueo.threading.CancellationToken;
import org.springframework.lang.NonNull;

public class BlobProviderGetArgs extends BlobProviderArgs
{
    public BlobProviderGetArgs(
        @NonNull String containerName,
        @NonNull BlobContainerConfiguration configuration,
        @NonNull String blobName,
        CancellationToken cancellationToken)
    {
        super(
                containerName,
                configuration,
                blobName,
                cancellationToken);
    }
}
